### PR TITLE
Cryptids

### PR DESCRIPTION
**Pull request by msgalenwhite**
Made a cryptid's table, model, controller, and show page - and tested them.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *= require_tree .
+ *= require_self
+ *= require foundation_and_overrides
+
+ */
+
+// @import "colors"
+
+.half-height{
+ min-height: 50vh
```
Comment (dpickett) [inline app/assets/stylesheets/application.scss]: I would use this unit sparingly as it isn't always reliable in older browsers

File: app/javascript/components/CurrentCrypto.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const CurrentCrypto = props => {
+
+
+  return(
+    <div id='CurrentCrypto'>
+      <img
+        className='homepageCryptoPic'
+        src='https://upload.wikimedia.org/wikipedia/commons/thumb/4/43/John_Oliver_November_2016.jpg/1200px-John_Oliver_November_2016.jpg'
```
Comment (dpickett) [inline app/javascript/components/CurrentCrypto.js]: license wise, I think you want to attribute wikipedia for this image - https://en.wikipedia.org/wiki/Wikipedia:Image_use_policy

File: app/javascript/components/MostRecentSightings.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react';
+
+class MostRecentSightings extends Component {
+  constructor(props){
+    super(props);
+    this.state={
```
Comment (dpickett) [inline app/javascript/components/MostRecentSightings.js]: recommend a space before and after the `=` in the assignment operation

File: db/migrate/20180425175958_create_cryptids.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CreateCryptids < ActiveRecord::Migration[5.2]
+  def change
+    create_table :cryptids do |t|
+      t.string :name, null: false
+      t.string :pic_url
+      t.text :description, null: false
+
+      t.belongs_to :user
```
Comment (dpickett) [inline db/migrate/20180425175958_create_cryptids.rb]: can a cryptid exist without a user? I think we want to require this field

File: spec/features/02_user_can_see_a_cryptid_show_page_spec.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+feature 'user can see a cryptid show page', %Q{
+  As a user
+  I want to see a single cryptid
+  To find out more about it
+} do
+  #Acceptance Criteria
+  # * If I go to the cryptid show page I see info about the cryptid
+
+  let!(:cryptid) {Cryptid.create!(
```
Comment (dpickett) [inline spec/features/02_user_can_see_a_cryptid_show_page_spec.rb]: multiline `let` statements should use a `do...end` block

File: spec/features/02_user_can_see_a_cryptid_show_page_spec.rb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  To find out more about it
+} do
+  #Acceptance Criteria
+  # * If I go to the cryptid show page I see info about the cryptid
+  let!(:test_user) { FactoryBot.create(:user) }
+  let!(:cryptid) do
+     Cryptid.create!(
+      name: "Dragon",
+      description: "Breathes fire",
+      user: test_user)
```
Comment (bkincart) [inline spec/features/02_user_can_see_a_cryptid_show_page_spec.rb]: This looks great! I'd just push the closing parentheses down to the next line in your next PR.

File: app/javascript/components/HighestRatedCryptid.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const HighestRatedCryptid = props => {
+  // let categories =['Aliens', 'Unicorns', 'Bigfoot', 'Lochness', 'Betty']
+  //
+  // let cryptids = categories.map((cryptid) => {
+  //   <li>{cryptid}</li>
+  // })
```
Comment (bkincart) [inline app/javascript/components/HighestRatedCryptid.js]: Typically, I won't want to merge comments - I'm assuming this is the next bit you're going to work on so I'll keep it here, but definitely note that we want to leave comments out of PRs unless we intend to keep them for clarification!

**Pull request merged**